app/assets/javascripts/discourse/app/controllers/activate-account.js

```diff
@@ -0,0 +1,79 @@
+import { tracked } from "@glimmer/tracking";
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
```

Comment:
the new `service` injection import! 🙂 

```suggestion
import { service } from "@ember/service";
```